core/src/models/App.ts

```diff
@@ -210,6 +215,7 @@ export class App extends LoggedModel<App> {
       pluginApp,
       createdAt: APIData.formatDate(this.createdAt),
       updatedAt: APIData.formatDate(this.updatedAt),
+      appRefreshQuery,
```

Comment:
This should be serialized:

```suggestion
      appRefreshQuery: await appRefreshQuery.apiData(),
```